backend/pom.xml

```diff
@@ -40,6 +86,35 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>org.openapitools</groupId>
+				<artifactId>openapi-generator-maven-plugin</artifactId>
+				<version>5.3.0</version>
+				<executions>
+					<execution>
+						<goals>
+							<goal>generate</goal>
+						</goals>
+						<configuration>
+							<inputSpec>${project.basedir}/src/main/resources/api.yaml</inputSpec>
+							<generatorName>spring</generatorName>
+							<addCompileSourceRoot>true</addCompileSourceRoot>
+							<modelPackage>${project.groupId}.${project.artifactId}.model.api</modelPackage>
```

Comment:
Nie wiem, jak dla mnie to bardziej by pasowało `models` zamiast `model`

Reply:
wszystko jedno to i tak folder

Reply:
*package, ale to i tak nie znaczy że nie można tego rozsądniej nazwać 🤷🏻